src/Passes/docs/src/UserGuide/QuickStart.md

```diff
@@ -0,0 +1,102 @@
+# Quick start
+
+Before we start, you will need to build the QAT tool. To this end, create a new folder `Debug` and type `make qat`:
+
+```sh
+mkdir Debug
+cd Debug
+make qat
+```
+
+A more detailed documentation of this step is available in the [build steps section](./BuildingLibrary.md). Once the build has been completed successfully, we will create a QIR in order to have an example code we can apply a profile to. The next step does not need to be completed if you already have a QIR. We will be using the Q# front end to generate the QIR and use the example `SimpleLoop`. However, you are free to choose another example in the `QirExamples` folder and/or another frontend. Go to the folder `./QirExamples/SimpleLoop/QSharpVersion` and type
+
+```sh
+make qir/Example.ll
+```
+
+This will generate a QIR that will have the path `./QirExamples/SimpleLoop/QSharpVersion/qir/Example.ll` relative to the project root.
+By using an application called QAT you can transform the QIR that was generated into a QIR that is tailored to a specific profile. Performing a transformation of the QIR from the `./Debug` folder is done by typing the following commands:
+
+```sh
+./Source/Apps/qat --apply --profile baseProfile -S ../QirExamples/SimpleLoop/QSharpVersion/qir/Example.ll
+```
+
+Validation of QIR profiles is not supported by the tool at the moment. We're working on this feature, and will add the quickstart documentation here when it is available.
+
+## Example: SimpleLoop
+
+For the sake of demonstration, we will look at the result of applying the profile to the previous Q# code and its corresponding QIR. However, instead of giving all the 3316 lines of the original QIR, we instead present the frontend code for the QIR. It is always possible to recreate the QIR from the step that was previously explained if you are curious:
+
+```
+namespace SimpleLoop {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Measurement;
+
+    function Value(r: Result): Int
+    {
+         return r == Zero ? 122 | 1337;
+    }
+
+    @EntryPoint()
+    operation Main(): Int
+    {
+        let nrIter = 5;
+        mutable ret = 1;
+        for _ in 1 .. nrIter {
+            use q = Qubit();
+            H(q);
+            let r = MResetZ(q);
+            set ret = Value(r);
+        }
+
+        return ret;
+    }
+}
+```
+
+The tool was ran with three options: `--apply`, `--profile baseProfile` and `-S`. First, it tells the tool to apply the profile to the QIR so that it can become a profile-specific QIR according to the profile selected. By specifying the value `baseProfile` to the argument `--profile`, we select which profile to use in the tool and the third argument ensures that LLVM IR will be printed to the terminal in a human readable format. The resulting code emitted (omitting declarations) is:
+
+```
```

Comment:
Did you try 
```
```ll
```

or 
```
```llvm
```

 here?

Reply:
I've added `ll` as this seems to be the right highlighter
```ll 
%0 = ; ... comment
````